src/lib/app/RvCommon/RvCommon/RvDocument.h

```diff
@@ -151,6 +157,7 @@ namespace Rv
         void changeEvent(QEvent*);
         bool event(QEvent*);
         void moveEvent(QMoveEvent*);
+        void resizeEvent(QResizeEvent*);
```

Comment:
Have you set clang-tidy for this repo? You should add override here:

```suggestion
        void resizeEvent(QResizeEvent*) override;
```